core/common/src/imodeljs-common.ts

```diff
@@ -149,6 +149,10 @@ export * from "./tile/TileMetadata";
  * @docs-group-description Geometry
  * Types for working with geometry.
  */
+/**
+ * @docs-group-description Serialization

```

Comment:
@pmconne @bbastings I added a new description section but this could also fall under the existing Geometry description.  Thoughts on changing it?

Reply:
Was waiting for @bbastings to chime in here.

Reply:
My preference is to keep serialization as that should match the flatbuffer stuff from geometry.

Reply:
Sounds good, I'll leave it as is then.